include/server_table.h

```diff
@@ -16,50 +16,6 @@
 #include <openssl/err.h>
```

Comment:
Do we anticipate this file containing anything in the immediate future? If not we should probably just get rid of it.

Reply:
I apologize that I wasn't more clear, I meant the whole file, not just that line.